src/cave.ui/PopupWidget.sling

```diff
@@ -177,3 +177,9 @@ func hidePopup virtual
 	}
 	anim.start()
 }
+
```

Comment:
This now needs to be updated to match the new way of doing this.

Reply:
Hi Sir. I have updated my pull request for setting of widget ID.